schema.json

```diff
@@ -4094,7 +4094,7 @@
               "name": "incomeAmountMonthly",
               "type": {
                 "kind": "SCALAR",
-                "name": "Float",
+                "name": "Decimal",
```

Comment:
Oh shoot, I think this is the `schema.json` from #2034...  it's an auto-generated file that we keep version controlled for various reasons.  I think the change from `Float` to `Decimal` is actually due to our upgrade of the `graphene-django` package, so it's possible you're using the new version of that package on this branch... so you might want to run `bash ./docker-update.sh` to revert to the old version of that package, and then run `python manage.py graphql_schema` to regenerate the `schema.json` so it looks like the version that's currently on `master`... if that makes sense.

Err alternatively though we could just merge #2034 and then these changes will be in-sync with `master`, hehe.

Reply:
ahaha, you're right - i was testing your django changes and forgot i had to revert back. will revert